otterdog/eclipse-zenoh.jsonnet

```diff
@@ -522,5 +522,17 @@ orgs.newOrg('eclipse-zenoh') {
         customRuleset("main"),
       ],
     },
+    orgs.newRepo('ci') {
+      allow_auto_merge: true,
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "GitHub Actions and workflows used across eclipse-zenoh",
+      secret_scanning_push_protection: "disabled",
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
we would like to have read permissions by default, you can still assign required permissions on a workflow basis.

Reply:
Do you wish to do this for all repositories or just this new one? The former is a breaking change and would naturally require changing all existing workflows.